docs/mast/mast.rst

```diff
@@ -142,7 +142,7 @@ To list data missions archived by MAST and avaiable through `astroquery.mast`, u
 
    >>> from astroquery.mast import Observations
    ...
-   >>> print(Observations.list_missions())
+   >>> print(Observations.list_missions())  # doctest: +IGNORE_OUTPUT
```

Comment:
This may be valuable to keep, as it shouldn't really change that often.

Reply:
OK, I was unsure about ones like this because the point of the function is to decouple the list of missions from the codebase so that when new missions are added the astroquery code doesn't have to be updated every time. But also, you are correct, this doesn't change that often.